CONTRIBUTING.md

```diff
@@ -57,14 +57,8 @@ You can add the sign off when creating the git commit via `git commit -s`.
 
 ## Code Reviews
 
-* Indicate the priority of each comment, following this
-[feedback ladder](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify/).
-If none was indicated it will be treated as `[dust]`.
-* A single approval is sufficient to merge, except when the change cuts
-across several components; then it should be approved by at least one owner
-of each component. If a reviewer asks for changes in a PR they should be
-addressed before the PR is merged, even if another reviewer has already
-approved the PR.
+* A single approval is sufficient to merge. If a reviewer asks for changes in a PR they should be
```

Comment:
We have some concept of component owners for compiler, but I don't know if it's enough to mention here. I can restore that text though if it makes sense

Reply:
agree with leaving it out.